daal4py/sklearn/cluster/_k_means_0_21.py

```diff
@@ -213,6 +214,7 @@ def fit(self, X, y=None, sample_weight=None):
                          np.allclose(sample_weight, np.ones_like(sample_weight)))
 
     if not daal_ready:
+        logging.info("kmeans:fit of sklearn")
```

Comment:
I think we need to improve wording. I offer to use this format:
1) sklearn.cluster.KMeans.fit: using original Scikit-learn...
2) sklearn.cluster.KMeans.fit: using Intel(R) DAAL...

but technically - LGTM too

Reply:
Agree with Egor. I also vote for creating a set of strings somewhere in common part of daal4py. Maybe in utils.py. The set of strings will look like:
    fit_method_uses_daal = “uses Intel® Data Analytics Acceleration Library solver”
    fit_method_uses_sklearn = “uses original Scikit-learn solver”
After that we can use those strings to generate log messages.
This will prevent us from the possible error when one log message has one wording an the another message - another wording.
It would also help to change the wording faster when needed,

Reply:
Agree.
Only one moment - I prefer to use ```Intel® DAAL``` instead of ```Intel® Data Analytics Acceleration Library```. Just because it's shorter.